
class ResultFailure {
  final Exception exception;

  ResultFailure(this.exception);
}
